From 8b2bc1fc922bfc3e26d199951bf6f99e882ca585 Mon Sep 17 00:00:00 2001
From: "Dr. David von Oheimb" <dev@ddvo.net>
Date: Fri, 13 Oct 2023 22:39:25 +0200
Subject: [PATCH 360/381] CMS_add1_signer.pod: add missing info on
 CMS_SignerInfo_sign() return values

Reviewed-by: Viktor Dukhovni <viktor@openssl.org>
Reviewed-by: Tomas Mraz <tomas@openssl.org>
(Merged from https://github.com/openssl/openssl/pull/22390)

(cherry picked from commit 2c581eb2fd92e997dfe0761b086eb690271cefc2)
---
 doc/man3/CMS_add1_signer.pod | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/doc/man3/CMS_add1_signer.pod b/doc/man3/CMS_add1_signer.pod
index 800085b7b8..11afecb905 100644
--- a/doc/man3/CMS_add1_signer.pod
+++ b/doc/man3/CMS_add1_signer.pod
@@ -31,8 +31,8 @@ Unless the B<CMS_REUSE_DIGEST> flag is set the returned CMS_ContentInfo
 structure is not complete and must be finalized either by streaming (if
 applicable) or a call to CMS_final().
 
-The CMS_SignerInfo_sign() function will explicitly sign a CMS_SignerInfo
-structure, its main use is when B<CMS_REUSE_DIGEST> and B<CMS_PARTIAL> flags
+The CMS_SignerInfo_sign() function explicitly signs a CMS_SignerInfo
+structure, its main use is when the B<CMS_REUSE_DIGEST> and B<CMS_PARTIAL> flags
 are both set.
 
 =head1 NOTES
@@ -90,6 +90,8 @@ before it is finalized.
 CMS_add1_signer() returns an internal pointer to the CMS_SignerInfo
 structure just added or NULL if an error occurs.
 
+CMS_SignerInfo_sign() returns 1 on success, 0 on failure.
+
 =head1 SEE ALSO
 
 L<ERR_get_error(3)>, L<CMS_sign(3)>,
-- 
2.34.1

